schemas/device/MRICoil.schema.tpl.json

```diff
@@ -0,0 +1,53 @@
+{
+  "_extends": "/core/schemas/products/device.schema.tpl.json",
+  "_type": "https://openminds.ebrains.eu/neuroimaging/MRICoil",
+  "properties": {
+    "activeElements": {
+      "_instruction": "If applicable, list the identifiers of the active coil elements or channels.",
+      "type": "string"
+    },
+    "coilGeometry": {
+      "_instruction": "Add the geometry of the coil (e.g., birdcage, loop, Helmholtz).",
+      "_linkedTypes": [
+        "controlledTerms:MRIRadiofrequencyCoilGeometry"
+      ]
+    },
+    "dimension": {
+      "_instruction": "Add the dimention of this coil",
+      "_linkedCategories": [
+        "mathematicalShapes"
+      ]
+    },
```

Comment:
Be aware of updated category (happened last week). It's now singular, i.e., "mathematicalShape"